setup.py

```diff
@@ -5,7 +5,7 @@
 setup(
     name='ebi_eva_common_pyutils',
     packages=find_packages(),
-    version='0.3.16',
+    version='0.3.17',
```

Comment:
Similar to our [modus operandi for Java projects](https://www.ebi.ac.uk/seqdb/confluence/display/VAR/Release+and+deployment+-+SOP), should we defer bumping versions until after the merge? This helps us iterate on the a snapshot version in PRs and makes version bumps explicit in the main branch commit history. I left a similar comment similar in another PR.

Reply:
I agree that maybe we should make version bumps as separate commits in Python projects (something I've not been really doing either...), just to make it clear in the commit history exactly when things are being released.

Somehow I haven't found snapshot versions as useful for Python projects, maybe just because it's so easy to just `pip install /path/to/other/project` to test things out... but it's probably also good practice and might facilitate better development and testing practices, so I'm happy to start doing this as well.

Reply:
I don't mind either way but maybe we should bump the version after a release so we know what we're working towards.

Reply:
That's an argument for using snapshot/development versions then - so this would be developed under `0.3.17-dev` (or `0.3.17.dev`, or `0.3.17-dev0`...) and merged.  Then we make a `bump version to 0.3.17` commit directly to the main branch, tag and release.  This way you know you're working towards `0.3.17` but you won't cause conflicting version names on pip.

Anyway, probably should be a (short) dev meeting topic!